src/plugins/vis_type_table/server/usage_collector/get_stats.ts

```diff
@@ -6,12 +6,16 @@
  * Side Public License, v 1.
  */
 
-import { ISavedObjectsRepository, SavedObjectsClientContract } from 'kibana/server';
 import {
-  SavedVisState,
-  VisualizationSavedObjectAttributes,
-} from 'src/plugins/visualizations/common';
-import { TableVisParams, VIS_TYPE_TABLE } from '../../common';
+  ISavedObjectsRepository,
+  SavedObjectsClientContract,
+  SavedObjectsFindResult,
+} from 'kibana/server';
+import { SavedVisState } from 'src/plugins/visualizations/common';
+import { VIS_TYPE_TABLE } from '../../common';
+
+// elasticsearch index.max_result_window default value
+const ES_MAX_RESULT_WINDOW_DEFAULT_VALUE = 1000;
```

Comment:
nit: This variable naming is confusing because [the ES default value for max result window is still 10k](https://www.elastic.co/guide/en/elasticsearch/reference/current/index-modules.html#index-max-result-window), even though you are changing the `perPage` size to 1000 here. I'd recommend renaming this.